README.md

```diff
@@ -1,13 +1,67 @@
 # Moxie
 [![Build Status](https://travis-ci.org/DarthStrom/Moxie.svg?branch=master)](https://travis-ci.org/DarthStrom/Moxie)
-
 A tasty mocking library for Swift
 
 ![Moxie](moxie.jpeg)
 
+## Creating mocks
+All mock objects must conform to the Mock protocol and have an instance of Moxie.
+
+```swift
+import Moxie
+
+class MockClass: Mock {
```

Comment:
That would be Swiftier! I've changed all but the one example to use protocols. Please let me know if I've gone too far.